tox.ini

```diff
@@ -10,6 +10,7 @@ deps =
   pytest-localserver
   urllib3
   certifi
+  requests
```

Comment:
Setup.py doesn't currently have any extras for urllib3 or requests, do you think we should?